hooks/pre-commit

```diff
@@ -13,3 +13,22 @@ then
   echo Please commit to a working branch and push it to GitHub for review/merge.
   exit 1
 fi
+
+readonly changed_files_command="git diff --cached --name-only | grep '\.py$' | grep -v 'servers/[^\/]*/jobs/models' | grep -v 'setup.py'"
```

Comment:
Why do you exclude these files, like setup.py? The list of files to exclude should be kept in sync with CircleCI. CircleCI only excludes ui/node_modules.